infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/ConnectorIaasController.java

```diff
@@ -191,7 +216,7 @@ public String createAzureInfrastructure(String infrastructureId, String clientId
     }
 
     public Set<String> createInstancesWithPublicKeyNameAndInitScript(String infrastructureId, String instanceTag,
-            String image, int numberOfInstances, int hardwareType, String publicKeyName, List<String> scripts) {
+            String image, int numberOfInstances, String hardwareType, String publicKeyName, List<String> scripts) {
```

Comment:
If now the parameter is  String hardwareType, then maybe it's not required any more:
String.valueOf(hardwareType) below?

Reply:
You're right. Fixed, thanks